aten/src/ATen/templates/NativeFunctions.h

```diff
@@ -31,3 +31,18 @@
 ${NativeFunctions_includes}
 
 ${NativeFunctions_declarations}
+
+namespace at {
+namespace native {
+
+// Added from torch/include/ATen/ops/mul_native.h
+struct TORCH_API structured_mul_out : public at::meta::structured_mul_Tensor {
```

Comment:
If you take a look at the built `NativeMetaFunctions.h` file, you'll see that we don't directly `#include <ATen/NativeMetaFunctions.h>` in it. So if you're going to defined the `at::meta::structured_mul_Tensor` struct in that file, you'll need to manually include that in this file too.